about.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet" />
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="about.css" />
+    <title>Document</title>
+</head>
+
+<body>
+    <div>
+        <ul class="social-nav">
+            <li><i class="fa fa-facebook" aria-hidden="true"></i></li>
+            <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </div>
+
+    <nav>
+        <a href="index.html"><img src="img/logo.png" alt="Logo" /></a>
+        <ul class="nav-menu">
+            <li class="nav-item">
+                <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Program</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Sponsor</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">News</a>
+            </li>
+        </ul>
+        <a class="nav-btn" href=""><span>CC COMPAIGN</span></a>
+        <div class="hamburger-menu">
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+        </div>
+    </nav>
+    <header>
+        <div class="main-section">
+            <h1>About</h1>
+            <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta doloremque officia enim adipisci sapiente expedita voluptas blanditiis animi aspernatur maxime ipsa provident quod odit beatae, ducimus excepturi explicabo aliquam architecto?
+            </p>
+        </div>
+    </header>
+    <main></main>
```

Comment:
- [ ] Kindly make sure to add a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo to your about page. 👌
- [ ] And a bottom section with a centered column that includes a section header, a divider, a line of text, and two images with text centered over them. 🙏
 ## Like in the design 👇️
![Screenshot from 2022-10-19 20-29-23](https://user-images.githubusercontent.com/89228743/196797611-85d31013-9954-473e-bea1-21badc33c76d.png)